server/py/services/api/migrations/tests/mysql/conftest.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2025 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+import os
+import sys
+
+import pytest
+import pytest_mock_resources
+
+mysql = pytest_mock_resources.create_mysql_fixture()
+
+logger = logging.getLogger("pytest_mock_resources")
+logger.setLevel(logging.DEBUG)
+
+handler = logging.StreamHandler(sys.stdout)
+handler.setLevel(logging.DEBUG)
+formatter = logging.Formatter("%(asctime)s %(levelname)s %(message)s")
+handler.setFormatter(formatter)
+
+logger.addHandler(handler)
```

Comment:
this is duplicated in `server/py/services/api/migrations/tests/base/conftest.py`